pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/PanDomainAuth.scala

```diff
@@ -17,7 +17,7 @@ import scala.concurrent.duration.FiniteDuration
 
 trait PanDomainAuth {
 
-  lazy val actorSystem = ActorSystem()
+  def actorSystem: ActorSystem
```

Comment:
I couldn't find a good reason for PanDomain to use its own actor system

Reply:
Now I have my answer. This was creating a new actor system for each controller, with one actor in it. A bit wasteful so there's a PR here to attempt to solve it: https://github.com/guardian/pan-domain-authentication/pull/41